main/ZsensorHCSR501.ino

```diff
@@ -33,6 +33,10 @@
 void setupHCSR501() {
   Log.notice(F("HCSR501 pin: %d" CR), HCSR501_GPIO);
   pinMode(HCSR501_GPIO, INPUT); // declare HC SR-501 GPIO as input
+#  ifdef HCSR501_LED_NOTIFY_PIN
+  pinMode(HCSR501_LED_NOTIFY_PIN, OUTPUT);
```

Comment:
also this macro should be added to the corresponding config file

Reply:
I added an example in the config_.h file (commented out)
`//#define HCSR501_LED_NOTIFY_GPIO 4 //Uncomment this line to mirror the state of the PIR sensor to the specified GPIO`